ui/src/app/pages/access/module-box.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { Clickable } from 'app/components/buttons';
+import { FlexRow } from 'app/components/flex';
+
+import { styles } from './data-access-requirements';
+
+export const ModuleBox = (props: {
```

Comment:
we could make the argument that we want every component to be its own file ... but I'm not 100% sure about that.  Here's an example where we might not want to.  This is only used by Module, so I might include this in Module.

Reply:
I agree. I moved ModuleBox inside of Module.